src/transformers/utils/index.js

```diff
@@ -1,6 +1,15 @@
 import fetch from 'node-fetch';
+import wrapFetch from 'fetch-retry';
 
-export const fetchOEmbedData = url => fetch(url).then(data => data.json());
+const fetchWithRetries = wrapFetch(fetch);
+
+export const fetchOEmbedData = url =>
+  fetchWithRetries(url, {
+    retries: 3,
+    retryDelay: attempt => {
+      return process.env.NODE_ENV === `test` ? 0 : 2 ** attempt * 1000; // 1000, 2000, 4000
```

Comment:
ok, this conditional here is causing coverage to drop - I didn't want for test cases that check failing network request to cause long test run, but I guess I need to implement it differently than conditional here